### ISSUE TITLE
When I use the --public-url option, There is a problems with 'https://'

### ISSUE THREAD

**Issue by limingyang2012** [parcel-bundler/parcel#409]
**Choose one:** is this a 🐛 bug report or 🙋 feature request?

### 😯 Current Behavior

my code is:

```js

  --public-url https://a.b.com/c.d.com/storage/projectname/

```

but I get: 

```js

  https:/a.b.com/c.d.com/storage/projectname/****

```

there is just one '/' after https:

### 🤔 Expected Behavior

I need get: 

```js

  https://a.b.com/c.d.com/storage/projectname/****

```

### 💁 Possible Solution

I think the code in `urlJoin.js` is wrong,

```js

const url = require('url');
const path = require('path');

module.exports = function(publicURL, assetPath) {
  // Use url.resolve to normalize path for windows
  // from \path\to\res.js to /path/to/res.js
  return url.resolve(path.join(publicURL, assetPath), '');
};

```

examples:

`url.resolve(path.join('https://www.baidu.com', '/df'), '')` => `https:/www.baidu.com/df`

`path.join('https://www.baidu.com', '/df')` => `https:/www.baidu.com/df`

`url.resolve('https://www.baidu.com', '/df')` => `https://www.baidu.com/df`

**Comment by 1120340041** [comment]
🐛

**Comment by brandon93s** [comment]
Hello @limingyang2012 

The `--public-url` is intended to be a path and not a fully qualified URI. The [documentation](https://parceljs.org/production.html#set-the-public-url-to-serve-on) could probably use some clarifications. This option is used to alter the resource path, relative to your domain (e.g. `--public-url ./storage/projectname`.

**Comment by 1120340041** [comment]
@brandon93s Sometimes we may have multiple domain names, and then we want to js and css assets into another domain name, so we want to  a fully qualified URI

**Comment by devongovett** [comment]
Should be fixed by #447.

**Comment by 1120340041** [comment]
@devongovett hi，parcel v1.4.1 is not solved this problem
